packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryResetPassword/index.test.tsx

```diff
@@ -256,14 +256,9 @@ describe('AccountRecoveryResetPassword page', () => {
       ).not.toBeInTheDocument();
 
       fireEvent.focus(newPasswordField);
-      await waitFor(
-        () => {
-          expect(screen.getByText('Password requirements')).toBeVisible();
-        },
-        {
-          timeout: SHOW_BALLOON_TIMEOUT,
-        }
-      );
+      await waitFor(() => {
+        expect(screen.getByText('Password requirements')).toBeVisible();
+      });
```

Comment:
FWIW, I tested that the element is indeed rendered and not a false positive by:
1) Running this page locally and observing the Password balloon when the field is in focus
2) Intentionally causing the test to fail by expecting `Password requirementz` to be visible.